python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/conftest.py

```diff
@@ -91,7 +101,9 @@ def pytest_configure(config: _pytest.config.Config):
             cudf_results = dict(read_results(f))
         config.stash[results] = (gold_results, cudf_results)
     else:
-        if "cudf.pandas" in config.option.plugins:
+        # Strip whitespace from each plugin name before checking
+        plugins = [plugin.strip() for plugin in config.option.plugins]
+        if "cudf.pandas" in plugins:
```

Comment:
It doesn’t seem like we use plugins anywhere else. Maybe we can just do this:
```suggestion
        if any(plugin.strip() == "cudf.pandas" for plugin in config.option.plugins):
```

Reply:
Done.